core/arch/arm/include/arm.h

```diff
@@ -76,6 +76,10 @@
 #define MPIDR_AARCH32_AFF_MASK	(MPIDR_AFF0_MASK | MPIDR_AFF1_MASK | \
 				 MPIDR_AFF2_MASK)
 
+#ifdef CFG_CORE_SEL2_SPMC
```

Comment:
I think we can remove the #ifdef, but please add a comment explaining what this is.

Reply:
Added comment as `/* MPIDR definitions for VCPU */`